language/move-prover/bytecode/src/compositional_analysis.rs

```diff
@@ -38,11 +38,7 @@ impl<'a> SummaryCache<'a> {
                 if fun_env.is_native_or_intrinsic() {
                     None
                 } else {
-                    Some(
-                        fun_data.annotations.get::<Summary>().expect(
-                            "Failed to resolve summary; recursion or scheduler bug suspected",
-                        ),
-                    )
+                    fun_data.annotations.get::<Summary>()
```

Comment:
Would be good to add a TODO here--this will avoid the crash but (I believe) will still not allow us to create summaries for recursive or mutually recursive code.

Reply:
done